apps/whale-api/src/module.indexer/rpc.block.provider.ts

```diff
@@ -88,9 +98,32 @@ export class RPCBlockProvider {
 
     const nextBlock = await this.client.blockchain.getBlock(nextHash, 2)
     if (await RPCBlockProvider.isBestChain(indexed, nextBlock)) {
+      this.logger.log(`[Synchronize - best chain] Indexing next block ${nextBlock.height}...`)
       await this.index(nextBlock)
     } else {
-      await this.invalidate(indexed.hash, indexed.height)
+      this.logger.log(`[Synchronize - not the best chain] Indexing prev block ${indexed.height}...`)
+      // Retry indexing the previous block before invalidating it
+      const MAX_RETRIES = 3
```

Comment:
we do not this ady after `REINDEX`

Reply:
Updated, then we will not be reindexing the previous block, just the current block thats failing. Also fine with me, we just have to test on any stucked ocean, to see if fix still works the same :)
cc: @pierregee